vmdb/app/assets/javascripts/miq_application.js

```diff
@@ -1229,11 +1236,11 @@ function miqBuildExplorerView(options) {
 
     // Only add header if option specified (because passing header:null still shows a thin header)
     if (settings.header != null) {
-      centerHash["header"] = settings.header
-    };
+      centerHash.header = settings.header;
+    }
 
     // Build the layout
-    if (settings.left == 0) {
+    if (!settings.left) {
```

Comment:
This change behaves in same way as before change:

Current:


    var a = 0;

    if (!a) {
      console.log("hello");
    } else {
      console.log("world");
    }

    => hello


Previous:


    var a = 0;

    if (a == 0) {
      console.log("hello");
    } else {
      console.log("world");
    }

    => hello